readme.md

```diff
@@ -56,10 +56,17 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
 - [7-Zip](http://www.7-zip.org/) (for deployment only)
 - [NSIS](http://nsis.sourceforge.net/) (for deployment only)
 
-### Mac:
+### macOS:
+
+- Xcode 8
+
+The program can also be built as a command line program using CMake. This type of build requires:
```

Comment:
The primary difference between the Xcode build and the CMake build is that the Xcode build creates a self-contained app bundle, whereas the CMake build just produces the binary. In general, the binaries are only meant to be run from the command line. Do you have any suggestions on how I can word this better?